dev/tasks/docker-tests/azure.linux.yml

```diff
@@ -52,7 +52,10 @@ jobs:
     displayName: Setup Archery
 
   - script: |
-      archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION="{{ arrow.no_rc_version }}" {{ run }}
+      archery docker run \
```

Comment:
You may try to use the following templating: https://github.com/apache/arrow/blob/master/dev/tasks/docker-tests/github.linux.yml#L42

Reply:
Or you can just switch to use the github actions template instead.

Reply:
OK!
I'll change the Azure Pipelines' template to use `flags`, `image` and `command` instead of `run`.